helper/namespace/namespace.go

```diff
@@ -64,14 +65,16 @@ func (n *Namespace) Validate() error {
 }
 
 const (
-	RootNamespaceID = "root"
+	RootNamespaceID   = "root"
+	RootNamespaceUUID = "00000000-0000-0000-0000-000000000000"
```

Comment:
FWIW, `RootNamespaceID` doesn't conform to ID requirements (Vault's were 5 characters base 62, we're at 6 ourselves anyways).

Reply:
Ah, sorry, the latter.

Reply:
I left this as-is for now @phyrog but we can revisit this later if we want.